src/common/logger.js

```diff
@@ -23,37 +21,60 @@ const shortenBrowserUrl = url => {
 const logLevels = {
   fatal: {
     rank: 1,
-    applyBrowserColor: text => `\u001b[31m${text}\u001b[0m`,
-    applyServerColor: text => `\u001b[31m${text}\u001b[0m`
+    serverLevelColor: '[31m',
+    serverMessageColor: '[0m',
+    browserLevelStyle: 'color: #f5222d; font-weight: bold',
+    browserMessageStyle: 'color: inherit'
   },
   error: {
     rank: 2,
-    applyBrowserColor: text => `\u001b[31m${text}\u001b[0m`,
-    applyServerColor: text => `\u001b[31m${text}\u001b[0m`
+    serverLevelColor: '[31m',
+    serverMessageColor: '[0m',
+    browserLevelStyle: 'color: #f5222d',
+    browserMessageStyle: 'color: inherit'
   },
   warn: {
     rank: 3,
-    applyBrowserColor: text => `\u001b[31m${text}\u001b[0m`,
-    applyServerColor: text => `\u001b[33m${text}\u001b[0m`
+    serverLevelColor: '[31m',
+    serverMessageColor: '[0m',
+    browserLevelStyle: 'color: #d46b08',
+    browserMessageStyle: 'color: inherit'
   },
   info: {
     rank: 4,
-    applyBrowserColor: text => `\u001b[34m${text}\u001b[0m`,
-    applyServerColor: text => `\u001b[34m${text}\u001b[0m`
+    serverLevelColor: '[34m',
+    serverMessageColor: '[0m',
+    browserLevelStyle: 'color: #096dd9',
+    browserMessageStyle: 'color: inherit'
   },
   debug: {
     rank: 5,
-    applyBrowserColor: text => `\u001b[30m${text}\u001b[0m`,
-    applyServerColor: text => `\u001b[37m${text}\u001b[0m`
+    serverLevelColor: '[30m',
```

Comment:
Tiny thing: could we please use the previous color, `[37m`, for the server? 🙏 
My eyes are not what they used to be 😅 

<img width="854" alt="Screen Shot 2022-02-23 at 18 28 27" src="https://user-images.githubusercontent.com/8189923/155374178-5a99b886-8a74-4886-95e0-3c9cfa2f683f.png">

Reply:
Oh, I accidentally took the browser color for the server (doesn't make a big difference on my terminal). It's corrected. Thanks for "seeing" it :-)

Reply:
😄 thanks